pkg/model/volumes.go

```diff
@@ -165,14 +170,36 @@ func (dev *Dev) PersistentVolumeEnabled() bool {
 // PersistentVolumeSize returns the persistent volume size
 func (dev *Dev) PersistentVolumeSize() string {
 	if dev.PersistentVolumeInfo == nil {
-		return OktetoDefaultPVSize
+		return dev.getDefaultPersistentVolumeSize()
 	}
 	if dev.PersistentVolumeInfo.Size == "" {
-		return OktetoDefaultPVSize
+		return dev.getDefaultPersistentVolumeSize()
 	}
 	return dev.PersistentVolumeInfo.Size
 }
 
+func (dev *Dev) isOktetoCloud() bool { // TODO: inject this
+	switch dev.Context {
+	case "https://cloud.okteto.com", "https://staging.okteto.dev":
```

Comment:
This is a code smell. Why do we need to know if it's cloud for a test? Could we validate based on a feature or something instead?

Reply:
I don't think we should be validating from the `dev.Context`. We can use `okteto.IsOktetoCloud `, that will check if it's okteto cloud or staging.

> This is a code smell. Why do we need to know if it's cloud for a test? Could we validate based on a feature or something instead?

We need to check if it's cloud/staging to keep the current volume sizes. If we update cloud and staging to use 5Gb the users will only be able to use one pv which will make their dev experience worse

Reply:
@jLopezbarb why wouldn't we check with the `Dev` struct?

Reply:
Dev is used to set `okteto.Context()` and there you have all the information about the context. Using `Dev` you only get the name but using `okteto.Context()` you have more information

Reply:
To have a unified single place where we do the check based on the context

Reply:
I have 2 problems with using `okteto.Context()`
- it causes a cyclic import. It could be addressed by a module re factorization in another pr
- we'd be depending on global variables from other packages, which could cause bugs and makes it harder to debug

I feel like the check could be injected though, because it makes sense that the `Dev` struct shouldn't know about the literal strings of staging and okteto

Reply:
Injecting the check could be a good option, do you think can be done without too many changes?

Reply:
@ifbyol @jLopezbarb There's not a simple way to inject the function because we'd end up in the same place with a circular import. 

There's a module refactor needed to fix this circular dependency, and I think it should be addressed in a follow up pr to [follow the MVC](https://github.com/okteto/okteto/blob/master/docs/code-review.md#pr-scope)

Reply:
ok, thanks for checking it!